app/code/Magento/InventoryInStorePickup/Test/_files/place_order.php

```diff
@@ -25,13 +23,4 @@
     ->create();
 $cart = current($cartRepository->getList($searchCriteria)->getItems());
 
-$product = $productRepository->get('SKU-1');
```

Comment:
I believe some of the older tests got affected by this change as they can't place order with empty cart.

Reply:
Fixed.
Will check with Travis results.